src/conv-core/cpuaffinity.C

```diff
@@ -415,8 +418,13 @@ typedef struct _affMsg {
 #endif
 } affMsg;
 
+static inline bool operator< (const skt_ip_t & a, const skt_ip_t & b)
```

Comment:
Instead of adding the function twice, could you put it into (e.g.) sockRoutines.h?

Reply:
I could, but I would need to add `#include <cstring>` (for memcmp) to that file. Would that be okay?

Reply:
Imho, that would be ok.

Reply:
Done